sklearn/decomposition/online_lda.py

```diff
@@ -154,7 +154,7 @@ class LatentDirichletAllocation(BaseEstimator, TransformerMixin):
     topic_word_prior : float, optional (default=None)
         Prior of topic word distribution `beta`. If the value is None, defaults
         to `1 / n_components`.
-        In the literature, this is called `beta`.
+        In the literature, this is called `eta`.
```

Comment:
This was changed from eta to beta in #10632. As far as I understand the original paper, beta is the word distribution (just like mentioned in this docstring BTW) and eta is indeed its prior. Just need additional input to be sure, @amueller ?

Reply:
This still just says "in the literature" which should be more specific

Reply:
Fixed in 7c46d34